### PR TITLE
Add base Table component

### PR DESCRIPTION
**Pull request by robinandeer**
## Justify why they are needed

Confirmation page.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #398** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/398" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/398?utm_source=stack-comment).

File: apps/store/src/components/Table/Table.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import styled from '@emotion/styled'
+
+type TableProps = { layout?: 'fixed' | 'auto' }
```
Comment (simonauner) [inline apps/store/src/components/Table/Table.tsx:3]: Should one of these be a default perhaps so one doesn't have to decide which one to use in every situation?

**Review by simonauner — APPROVED** [review]
Nice with something to build on. Just had one thought that needs to be decided now, but perhaps as a question of component API design - do we want sane (some) defaults?

File: apps/store/src/components/Table/Table.tsx (hunk 1 above)
Comment (robinandeer) [reply apps/store/src/components/Table/Table.tsx:3]: "auto" is already the default in browsers

https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout

Comment (simonauner) [reply apps/store/src/components/Table/Table.tsx:3]: Oh, sorry, didn't spot the  `? ` and it was `fixed` in both story examples. Good 👍

**Pull request merged**